`doc/Makefile` fails if path to repo contains spaces

### Godot version

Godot repo @ 461037203cc0ec4f8a59422163b37e29e125d5fc

### System information

Arch Linux (IDK how to give a version for this)

### Issue description

If you clone the Godot repo into a path that contains spaces, then you’ll get errors when you try to use `doc/Makefile`:

<pre><samp>$ make rst
rm -rf /tmp/tmp.gYcFoxDYn1/Godot repo/doc/_build/rst
mkdir -p /tmp/tmp.gYcFoxDYn1/Godot repo/doc/_build/rst
python3 /tmp/tmp.gYcFoxDYn1/Godot repo/doc/tools/make_rst.py -o /tmp/tmp.gYcFoxDYn1/Godot repo/doc/_build/rst -l en /tmp/tmp.gYcFoxDYn1/Godot repo/doc/classes/ /tmp/tmp.gYcFoxDYn1/Godot repo/doc/../modules/
/usr/bin/python3: can't find '__main__' module in '/tmp/tmp.gYcFoxDYn1/Godot'
make: *** [Makefile:20: rst] Error 1
</samp></pre>

### Steps to reproduce

1. Create a folder named “Godot repo”.
2. Put a copy of the Godot repo into that folder.
3. Change directory into the `Godot repo/doc`
4. Run `make rst`

### Minimal reproduction project

N/A

--- a/doc/Makefile
+++ b/doc/Makefile
@@ -1,4 +1,4 @@
-BASEDIR = $(CURDIR)
+BASEDIR = .
 CLASSES = $(BASEDIR)/classes/ $(BASEDIR)/../modules/
 OUTPUTDIR = $(BASEDIR)/_build
 TOOLSDIR = $(BASEDIR)/tools

